lib/suseconnect_register.pm

```diff
@@ -39,6 +51,13 @@ sub suseconnect_registration {
     zypper_call 'lr';
 }
 
+=head2 command_register
+
+ command_register();
```

Comment:
This does not list the function args.

It should look like this: `command_register($version, $addon, $addon_regcode);`

Also make sure to describe each variable in the description text. And write the vars like this `C<$var>` for nice formatting.